tutorials/polkadot-sdk/parachains/deploy-to-relay-chain/deploy-on-paseo.md

```diff
@@ -14,18 +14,22 @@ This tutorial demonstrates deploying a parachain on a public test network like t
 Before you start, you need to have the following prerequisites:
 
 - You know how to generate and modify chain specification files as described in the [Generate Chain Specs](/develop/parachains/deployment/generate-chain-specs){target=\_blank} section
+- You have installed the `polkadot-omni-node` and `chain-spec-builder` binaries
```

Comment:
What do you think about adding these references?
- https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html
- https://docs.rs/staging-chain-spec-builder/latest/staging_chain_spec_builder/